addons/base_vat/models/res_partner.py

```diff
@@ -292,6 +292,27 @@ def check_vat_al(self, vat):
             return True
         return False
 
+    __check_tin1_ro_natural_persons = re.compile(r'[1-9]\d{2}(0[1-9]|1[0-2])(0[1-9]|[12]\d|3[01])\d{6}')
+    __check_tin2_ro_natural_persons = re.compile(r'9000\d{9}')
```

Comment:
I think we're wrong here.
The documentation is saying that we have 13 characters in the first time but only provide the rule for 12 characters. And the value you put in `_ref_vat` contains only 12 characters. 
If you try your regex on regex101.com, here is the result:
![image](https://github.com/odoo/odoo/assets/6582885/114397be-0c52-4fb1-a2f2-38e45fa17975)
The example in the code is wrong, we have to add one digit.

Reply:
Yeah from other website the correct value is 13, i just forgot to change the ref_vat when i saw that the documentation was faulty, thanks ! :smile: